*: Fix old appc/spec version in various files

krnowak opened the pull request:
~~It is outdated compared to the acVersion in the ACI manifest of the coreos flavor.~~

There are more places where outdated appc/spec version is used. This PR replaced all of them and adds a new script for automating it. It could be used right after vendoring in the new version of appc/spec.

lucab commented:
[Rest of rkt](https://github.com/coreos/rkt/blob/master/glide.yaml#L22:L23) is at 0.8.5, perhaps we should bump them all here?

krnowak commented:
Updated. I replaced all the old appc/spec version with 0.8.5 and added a script for automating that.

Might be good to put somewhere some information about updating appc/spec to the new version, like:

- vendor the new version of appc/spec with glide
- run `./scripts/bump-appc-spec-version
- commit

krnowak commented on Documentation/app-container.md:
```diff
@@ -33,8 +33,8 @@ To validate that `rkt` successfully implements the ACE part of the spec, use the
 	--volume=database,kind=host,source=/tmp \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-main.aci \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-sidekick.aci
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-main.aci \
```
These links need to be updated manually, so I'm printing a message in the `bump-appc-spec-version` script about it.

krnowak commented on Documentation/app-container.md:
```diff
@@ -33,8 +33,8 @@ To validate that `rkt` successfully implements the ACE part of the spec, use the
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-main.aci \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-sidekick.aci
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-main.aci \
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-sidekick.aci
```
These links need to be updated manually, so I'm printing a message in the `bump-appc-spec-version` script about it.

lucab replied:
It looks like those were simply missing in latest releases, @jonboulle just uploaded them for [0.8.6](https://github.com/appc/spec/releases/tag/v0.8.6), so it should be fine next time we bump appc.

krnowak replied:
I just remembered that they were missing actually quite often.

lucab replied:
This is being taken care of: https://github.com/appc/spec/pull/646
0.8.5 and 0.8.6 artifacts are now both in place. May you reconsider?

krnowak replied:
Ah, cool! Sure, will do, thanks.

krnowak commented:
Updated, fixed the links to the ACE validator images and dropped the message from the bump-appc-spec-version script.

lucab commented:
LGTM
